sdk/extensions/Microsoft.Extensions.Azure.WebJobs/samples/Function1.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Azure.Storage.Blobs;
+using System.Linq;
+
+namespace Azure.Extensions.WebJobs.Sample
+{
+    #region Snippet:AzureClientInFunction
+    public static class Function1
+    {
+        [FunctionName("Function1")]
+        public static IActionResult Run(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", "post", Route = null)] HttpRequest req,
+            [AzureClient("AzureWebJobsStorage")] BlobServiceClient client)
```

Comment:
Do you think for the sample it would be worthwhile to use a different name than `AzureWebJobsStorage`?  I think that if you try to set that to an object it's going to break a bunch of other stuff.  I get what you are going after here, I'm just not sure if folks will understand the implication of this specific name and how it's used by other parts of the system that are not yet enlightened to handle the ability to set an object here?

Reply:
My intent was mostly to reuse the connection string most functions already have but you are right that it might be beneficial to stay away from touching it for now.

Reply:
Your call, might be nice to add a small note to the README about the implications of changing this specific value if we continue with this pattern.